connPFM/tests/test_ev.py

```diff
@@ -16,7 +16,7 @@ def test_calculate_ets(ets_auc_original_file, AUC_file, atlas_file):
 
     AUC_img = masker.fit_transform(AUC_file)
     ets_auc_orig = np.loadtxt(ets_auc_original_file)
-    ets_auc_calc, u_vec, v_vec = ev.calculate_ets(AUC_img, AUC_img.shape[1])
```

Comment:
if you are moving the the `calculate _ets` function to `utils`, you should also move its testing to `test_utils.py` let's be coherent. Same goes for all the other moved functions

Reply:
This is a different utils (it's inside the connectivity folder). We could create a `test_connectivity_utils.py` if that's better in your opinion.